docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,268 @@
+  nonce : opt nat64;
+};
+```
+Specify by `perpercentage_to_spawn` the portion of the maturity that should be spawn into a neuron in percentage. This should be a value between 1 and 100 (inclusive). 
+If the spawned neurons should have a different controller than the parent neuron (from which you spawn the maturity), you can optionally define a different controlling principal by `new_controller`. Otherwise, the spawned neuron will have the same controller as the parent neuron. 
+Similarly to claiming a neuron (see above), you need to specify the new neuron's ledger address and choose a nonce for this in the last argument `nonce`.
+Based on this nonce and the controlling principal, the neuron's address is uniquely defined (see section _Compute the neuron's address_).
+
+_Required permissions:_
+Only a neuron's controller can spawn maturity from a neuron.
+
+### Disbursing / unstaking a neuron
+When a neuron is dissolved, i.e., its dissolve delay is zero, the neuron's controlling principal can instruct it to _disburse_ the neuron’s stake. 
+This means that the staked ICP balance is transferred to a specified new ledger account and is liquid again.
+The following command can be used to disburse a neuron. 
+```
+type Disburse = record {
+  to_account : opt AccountIdentifier;
+  amount : opt Amount;
+};
+```
+The command takes as argument the ledger account `to_account` to which the ICP tokens should be transferred to and the amount `amount` that should be disbursed. The latter allows to only disburse a portion of a neuron's stake.
+
+_Required permissions:_
+Only a neuron's controller can disburse a neuron.
+
+
+### Voting with a neuron
+A neuron represents a participant of the NNS DAO and can participate in governance. Concretely, a neuron can submit proposals or vote on them if its dissolve delay is at least 6 months. 
+A neuron can either directly vote to adopt or reject a given proposal or it can _follow_ other neurons for certain decisions - which will automatically cast a ballot for the neuron if enough of the followees voted.
+
+**Vote directly**
+To vote directly, first find out what proposals can be voted on and then find the ID of the proposal you want to vote on.
+The following are the relevant NNS governance API and record that can also be found in the candid file.
+```
+service : (Governance) -> {
+  get_pending_proposals : () -> (vec ProposalInfo) query;
+};
+type ProposalInfo = record {
+  id : opt NeuronId;
+  ...
+};
+ ```
+First use the `get_pending_proposals` API to get the `ProposalInfo` of all proposals that are still pending. 
+Then, as part of this proposal information you can find a proposal's id `id`.
+
+Using this, you can now vote on a proposal using the following neuron command.
+```
+type RegisterVote = record { 
+  vote : int32; 
+  proposal : opt NeuronId 
+};
+```
+The `vote` represents if the neuron should adopt or reject the proposal and the `proposal` is the proposal ID of the proposal that the neuron votes on.
```

Comment:
What is Yes and what is no? How can a user find out?

Reply:
It is defined here: https://gitlab.com/dfinity-lab/public/ic/-/blob/7d17d2785317c6c19ecd00599872461154acf4a1/rs/nns/governance/proto/ic_nns_governance/pb/v1/governance.proto#L427-437

```
  VOTE_YES = 1;
  VOTE_NO = 2;
  ```

Reply:
I don't know how a user would normally be expected to find this out